hack/deploy_fast.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+check_ocp() {
+    local KUBECONFIG_PATH=$1
+
+    export KUBECONFIG=$KUBECONFIG_PATH
+    
+    if oc get nodes; then
+        return 0
+    else
+        return 1
+    fi
+}
```

Comment:
We don't know if this is the right cluster. This is ok for now, but it's not really fool-proof.